cmake/CreateBazelConfig.cmake

```diff
@@ -115,13 +115,14 @@ function (export_list_to_bazel filename VAR)
         APPEND "${filename}"
         [=[
 """Automatically generated unit tests list - DO NOT EDIT."""
```

Comment:
Nit: Perhaps the "unit tests" string should be an `export_list_to_bazel` parameter.

Reply:
I will fix that in a follow-up PR, it changes dozens of files unrelated to the current problem.